docs/documentation/maintenance-coordinator.md

```diff
@@ -0,0 +1,91 @@
+# Coordinator Model
+
+## Table of Contents
+
+<!-- MarkdownTOC depth=0 autolink="true" bracket="round" autoanchor="true" style="ordered" indent="   " -->
+1. [Role of the Coordinator](#role-of-a-coordinator)
```

Comment:
Wouldn't it be good to add chapter of becoming a coordinator? I'm currently missing this part, but maybe somone can enlighten me :+1:

Also who decides demote the person from being a coordinator?

Reply:
Hold up your hand if one is looked for ;-). Currently we do not expect struggling with to many willing coordinators. However you are right. We should lose a word or two over this. In my understanding, the PM would make the final decision in case of some argument. Note that is my personal understanding of this matter, we did not discuss this in the TB.

Reply:
Any concrete proposal for the wording?